howto/setup-state-store/setup-azure-cosmosdb.md

```diff
@@ -75,13 +101,15 @@ kubectl apply -f cosmos.yaml
 
 ### Running locally
 
-To run locally, create a `components` dir containing the YAML file and provide the path to the `dapr run` command with the flag `--components-path`.
+To run locally, create a YAML file described above and provide the path to the `dapr run` command with the flag `--components-path`.  See [this](https://github.com/dapr/cli#use-non-default-components-path) or run `dapr run --help` for more information on the path.
 
 ## Partition keys
 
-The Azure CosmosDB state store will use the `key` property provided in the requests to the Dapr API to determine the partition key.
+For **non-actor** state operations, the Azure CosmosDB state store uses the `key` property provided in the requests to the Dapr API to determine the partition key.  This can be overridden by specifying a metadata field in the request with a key of `partitionKey` and a value of the desired partition.
+
```

Comment:
Can you provide an example of this "by specifying a metadata field in the request with a key of `partitionKey` and a value of the desired partition." to show how this metadata can be used

Reply:
Reorganized this section and added metadata with partitionKey examples.